s3/replication/common/src/s3replicationcommon/s3_update_replication_status.py

```diff
@@ -0,0 +1,238 @@
+        full_uri = self._session.admin_endpoint + request_uri
+
+        query_params = ""
+        body = value or ""
+        headers = AWSV4Signer(
+            self._session.admin_endpoint,
+            self._session.service_name,
+            self._session.region,
+            self._session.access_key,
+            self._session.secret_key).prepare_signed_header(
+            'GET' if value is None else 'PUT',
+            request_uri,
+            query_params,
+            body
+        )
+
+        if (headers['Authorization'] is None):
+            self._logger.error(fmt_reqid_log(self._request_id) +
+                               "Failed to generate v4 signature")
+            sys.exit(-1)
+
+        self._logger.info(fmt_reqid_log(self._request_id) +
+                          'Motr index operation on {} {}'.format(
+                              full_uri, body))
+
+        if value is None:
+            return self._session.get_client_session().get(
+                full_uri,
+                params=query_params, headers=headers)
+        else:
+            return self._session.get_client_session().put(
+                full_uri,
+                params=query_params, headers=headers,
+                data=body.encode())
+
+    async def update(self, status):
+        self._timer.start()
+        self._state = S3RequestState.RUNNING
+
+        try:
+            if self._session.admin_endpoint is None:
+                self._logger.warn(fmt_reqid_log(self._request_id)
+                                  + 'Admin API not configured, '
+                                  + 'skipping source metadata update')
+                self._state = S3RequestState.COMPLETED
+                return
+
+            async with self.kv_session(
+                    self.global_bucket_index_id,
+                    self._bucket_name) as resp:
+
+                if resp.status == 200:
+                    bucket_info = json.loads(await resp.text())
+                    self._logger.info(fmt_reqid_log(self._request_id) +
+                                      'Bucket index lookup for {}'.
+                                      format(self._bucket_name) +
+                                      'response received with' +
+                                      ' status code: {}'.
+                                      format(resp.status))
+                    self._logger.debug(
+                        'bucket info: {}'.format(bucket_info))
+
+                else:
+                    self._state = S3RequestState.FAILED
+                    error_msg = await resp.text()
+                    self._logger.error(
+                        fmt_reqid_log(self._request_id) +
+                        'Index operation failed with http status: {}'.
+                        format(resp.status) +
+                        ' Error Response: {}'.format(error_msg))
+                    return
+
+            bucket_owner = bucket_info['account_id']
```

Comment:
https://github.com/Seagate/cortx-multisite/blob/main/s3/replication/manager/src/s3replicationmanager/prepare_job.py - Here, we get 'Owner-Account-id' from fdmi template. 
You can update the code.